play/src/pages/gameinprogress/DiscussAnswer.tsx

```diff
@@ -89,6 +102,28 @@ export default function DiscussAnswer({
             answerIndex={correctIndex ?? 0}
             currentState={currentState}
           />
+          {isSmallDevice && currentState === GameSessionState.PHASE_1_DISCUSS && (
+          <Typography
+            variant="body1"
+            sx={{
+              textAlign: 'center',
+              marginTop: `${theme.sizing.largePadding}px`,
+              opacity: 0.5,
+            }}
+          >
+            <pre
```

Comment:
What's happening with the use of the `pre` component here?

Reply:
@drewjhart

Reply:
so `white-space` is the CSS tag that can really help here. You can use it to recognize `\n` if you need a line break or just to recognize two different components as being on separate lines. You can find two implementations as case studies in our code:

`play/.../QuestionCard.tsx`:
```
  <Typography
          variant="body1"
          sx={{ fontWeight: 700, whiteSpace: 'pre-line' }}
        >
          {`\n ${questionText[1]}`}
        </Typography>
```

`play/.../pregameCountdown.tsx`:
```
const TypographyStyled = styled(Typography)(({ theme }) => ({
  textAlign: `center`,
  paddingLeft: `${theme.sizing.mediumPadding}px`,
  paddingRight: `${theme.sizing.mediumPadding}px`,
  whiteSpace: 'pre-wrap',
}));
```

There are some differences between `pre-wrap` and `pre-line` so take a look at the docs but this should allow you to remove the pre tags and just get something that is like <Typography ...> text </typography>

Reply:
fixed